src/core/chatSession.js

```diff
@@ -144,6 +145,14 @@ const CHAT_SESSION_FACTORY = new PersistentConnectionAndChatServiceSessionFactor
 
 var setGlobalConfig = config => {
   var loggerConfig = config.loggerConfig;
+  /**
+    * if config.loggerConfig.logger is present - use it in websocketManager
+    * if config.loggerConfig.customizedLogger is present - use it in websocketManager
+    * if config.loggerConfig.useDefaultLogger is true - use default window.console + default level INFO
```

Comment:
So Advanced_Log has priority set to 50 -- 
which makes it will always be published by default assuming Customer have `error`, `warn` or any logLevel enabled.